### ISSUE TITLE
Admin portal still uses login domain to load resources

### ISSUE THREAD

**Issue by solosky** [keycloak/keycloak#13665]
### Describe the bug

For security consideration, and following the recomendation in  https://www.keycloak.org/server/reverseproxy, I plan to expose 2 endpoints by different domain:

## kcadmin.example.com

This domain will be the admin portal endpoint, admin user will access this endpoint to manage keycloak by admin console. It should be retricted access with internal network. 
Those paths are forwarded to origin: /*

## login.example.com

This domain will be the login portal for end user, the endpoint should expose to public user, and no restriction access from public internet. 
Those paths are forwarded to origin: /js/*, /realms/*, /resources/*, /robots.txt, 

## Configuration
There is some modification in keycloak.conf to fit that setting, here is the key configuration in keycloak.conf. 

```
proxy=edge
hostname=login.example.com
hostname-admin=kcadmin.example.com
```
also started the KC with ```kc.sh start``` command. 
 

### Version

docker container image: quay.io/keycloak/keycloak:19.0.1

### Expected behavior

The admin console should be works as expected access by domain kcadmin.example.com. 

### Actual behavior

The admin console can not be display correctly.
The reason is the admin console stil request admin resources under /admin/* by login.example.com, which is not be forwarded to origin by its forwarding rule. 

The resources /admin/* should be requested with domain kcadmin.example.com

Although we can add the /admin/* forward rule to login.example.com, it stil has serious security issue because the admin console will be exposed to public internet network. 

### How to Reproduce?

It's very easy to reproduce the issue by setup correctly acorrding the description. 

### Anything else?

Admin resources requested by domain login.example.com 
<img width="963" alt="企业微信截图_20220809163959" src="https://user-images.githubusercontent.com/480111/183604959-520c0426-c4ee-4621-9a31-34ab890e5ccc.png">

**Comment by tim-hutchinson** [comment]
This looks to be the same as https://github.com/keycloak/keycloak/issues/12747 and also affects our environment, where we only block admin paths from internet clients. Immediately after authenticating to the admin console, the `whoami`, `realms`, and `serverInfo` requests are all sent to the login hostname, and fail.

From previous testing, this also occurred in Keycloak 18.0.X versions and prevents us from using the new Admin UI

**Comment by tobilarscheid** [comment]
I see the exact same issue here on a fresh deployment of `19.0.1` in k8s. Current solution looks like this `KC_FEATURES_DISABLED=admin2` for me....

**Comment by Ali-Javanmardi** [comment]
I also have same experience with 19.0.1 docker container.
I can access my admin root address and as soon as I try to login it redirects to main hostname/realms and after login it returns to admin hostname but some resources in page try to access main hostname in their address.
Seems completely similar to what @solosky  described in first post.

**Comment by stianst** [comment]
We believe this is fixed in https://github.com/keycloak/keycloak-ui/pull/3373

Could you please try the nightly release (quay.io/keycloak/keycloak:nightly) and check if that works?

**Comment by solosky** [comment]
> We believe this is fixed in [keycloak/keycloak-ui#3373](https://github.com/keycloak/keycloak-ui/pull/3373)
> 
> Could you please try the nightly release (quay.io/keycloak/keycloak:nightly) and check if that works?

Sorry to reply late.

I have tested the new version 19.0.3, and confirmed that this issue is resolved now.

Thank your very much on this.
